liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -959,4 +854,110 @@ public void info(String msg) {
 
     }
 
+    private Plugin getLibertyPlugin() {
+        Plugin plugin = project.getPlugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID + ":" + LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID);
+        if (plugin == null) {
+            plugin = plugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID, LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID, "LATEST");
+        }
+        return plugin;
+    }
+
+    private Xpp3Dom getLibertyPluginConfig() {
+        // get the Liberty plugin configuration from the pom and overrides looseApplication to true.
+        // ignores the plugin execution configuration just like CLI invocation (e.g. mvn liberty:create) ignoring execution configuration.
+        Plugin libertyPlugin = getLibertyPlugin();
+        Xpp3Dom overrides = configuration(element(name("looseApplication"), "true"));
+        return getPluginConfig(libertyPlugin, overrides);
+    }
+
+    private Xpp3Dom getPluginConfig(Plugin plugin, Xpp3Dom overrides) {
+        Xpp3Dom config = (overrides != null) ? overrides : configuration();
+        Xpp3Dom pluginConfig = (Xpp3Dom)plugin.getConfiguration();
+        if (pluginConfig != null) {
+            config = Xpp3Dom.mergeXpp3Dom(config, pluginConfig);
+        }
+        return config;
```

Comment:
Does this include configuration from within goal-specific `<execution>` elements such as in [these](https://github.com/OpenLiberty/ci.maven/blob/master/docs/deploy.md) examples?

Reply:
No it doesn't and I have already added the comment in `getLibertyPluginConfig()` method.  If you think we will need it, we need to update this method.  
```
    private Xpp3Dom getLibertyPluginConfig() {
        // get the Liberty plugin configuration from the pom and overrides looseApplication to true.
        // ignores the plugin execution configuration just like CLI invocation (e.g. mvn liberty:create) ignoring execution configuration.
        Plugin libertyPlugin = getLibertyPlugin();
        Xpp3Dom overrides = configuration(element(name("looseApplication"), "true"));
        return getPluginConfig(libertyPlugin, overrides);
    }
```